.github/workflows/ci.yml

```diff
@@ -202,6 +202,124 @@ jobs:
           ~/.cargo/git/db/
         key: cargo-${{ matrix.toolchain }}-${{ matrix.platform.name }}-${{ hashFiles('Cargo.lock') }}
 
+  tier2:
+    name: Check winit for Tier 2 Targets
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        toolchain: [stable, nightly, '1.73']
+
+    steps:
+    - uses: taiki-e/checkout-action@v1
+
+    - uses: dtolnay/rust-toolchain@master
+      with:
+        toolchain: ${{ matrix.toolchain }}
+
+    - name: Use specific dependency versions for Rust 1.63 compatibility.
+      if: matrix.rust == '1.63'
+      run: |
+        cargo update --package=dashmap --precise 5.4.0
+        cargo update --package=regex --precise=1.9.0
+        cargo update --package=half --precise=2.2.1
+
+    - run: >
+        rustup target add
+        x86_64-unknown-linux-musl
+        x86_64-unknown-linux-gnux32
+        x86_64-linux-android
+        i686-linux-android
+        x86_64-unknown-freebsd
+        x86_64-unknown-netbsd
+        x86_64-unknown-illumos
+        i686-unknown-linux-musl
+        riscv64gc-unknown-linux-gnu
+        aarch64-unknown-linux-gnu
+        aarch64-unknown-linux-musl
+        powerpc64le-unknown-linux-gnu
+        armv5te-unknown-linux-gnueabi
+        s390x-unknown-linux-gnu
+        arm-linux-androideabi
+        sparc64-unknown-linux-gnu
+        sparcv9-sun-solaris
+
+    - run: cargo check --release -vv --all-targets --target=x86_64-unknown-linux-musl
+    - run: cargo check --release -vv --all-targets --target=x86_64-unknown-linux-musl --no-default-features --features x11
```

Comment:
- Why are we using `--release`?
- The output of `-vv` seems to make the outout quiet hard to parse to me.
- Why `--all-targets`? What exactly are we testing in this case?